workshop/Lab1/README.md

```diff
@@ -1,17 +1,17 @@
 # Lab 1. Set up and deploy your first application
 
 Learn how to deploy an application to a Kubernetes cluster hosted within
-the IBM Container Service.
+the IBM Cloud Kubernetes Service.
 
-# 0. Install Prerequisite CLIs and Provision a Kubernetes Cluster
+## 1. Install prerequisite CLIs and provision a Kubernetes cluster
 
 If you haven't already:
 1. Install the IBM Cloud CLIs and login, as described in [Lab 0](../Lab0/README.md).
 2. Provision a cluster:
```

Comment:
I assume that devs with a linux background wouldn't need further instructions on this?